Sources/Evangelium/Helpers/ReadingFactory.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  ReadingFactory.swift
+//  Evangelium
+//
+//  Created by Manuel Sánchez on 2/4/20.
+//
+
+import Foundation
+
+struct ReadingFactory {
+    static func create(from todayReadings: [Reading], with date: String) -> SwiftyReading{
+        let readings = todayReadings.filter { reading in reading.type == "reading" }
+        let psalm = todayReadings.filter { reading in reading.type == "psalm" }
+        let gospel = todayReadings.filter { reading in reading.type == "gospel" }
+        
+        if readings.count == 2 {
```

Comment:
you could try using pattern matching here, it could make this a little bit more readable.

Reply:
You could store the count of the array in a constant in order to avoid call `count` property of the array on each if statement.